haystack/nodes/preprocessor/preprocessor.py

```diff
@@ -829,6 +855,43 @@ def _split_sentences(self, text: str) -> List[str]:
         sentences = sentence_tokenizer.tokenize(text)
         return sentences
 
+    def _split_tokens(self, text: str, tokenizer: Any) -> List[str]:
+        if tokenizer == "tiktoken":
+            try:
+                import tiktoken
+            except ImportError:
+                raise ImportError(
+                    "Tiktoken is needed in order to split documents by tokens. "
+                    "Please install it with `pip install tiktoken`."
+                )
```

Comment:
Let's handle this optional import at the top, just like with transformers.

Here we'll need only `tiktoken_import.check()`